### PR TITLE
adding an example of update release

### PR DESCRIPTION
**Pull request by mw-hrastega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,8 +128,8 @@ When you define your workflow in the `.github/workflows` directory of your repos
...
+| `release` | <p>(Optional) MATLAB release to set up. You can specify R2021a or a later release. By default, the value of `release` is `latest`, which corresponds to the latest release of MATLAB.</p><p><ul><li>To set up the latest update of a release, specify only the release name, for example, `R2023b`.</li><li>To set up a specific update release, specify the release name with an update number suffix, for example, `R2023bU4`.</li><li>To set up a release without updates, specify the release name with an update 0 or general release suffix, for example, `R2023bU0` or `R2023bGR`.</li></ul></p><p>**Example**: `release: R2023b`<br/>**Example**: `release: latest`<br/>**Example**: `release: R2023bU4`</p>
```
Comment (mcafaro) [inline README.md:131]: The "To set up a release without updates..." line could be left out to make this a little more succinct. That seems pretty edge-casey and those details are on the MPM page.

**Review by mcafaro — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mcafaro) [reply README.md:131]: Nevermind, I didn't realize this was bulleted. As a bulleted list, it seems better to leave in. I was reading it as a giant paragraph.

Comment (mw-hrastega) [reply README.md:131]: Sure, Mark.

**Review by yitingw3 — APPROVED** [review]
Looks good to me

**Review by davidbuzinski — APPROVED** [review]

**Pull request merged**